internal/pkg/deploy/cloudformation/stack/testdata/workloads/svc-staging.stack.yml

```diff
@@ -117,6 +117,7 @@ Resources: # If a bucket URL is specified, that means the template exists.
     Type: AWS::IAM::Role
     Properties:
       AssumeRolePolicyDocument:
+        Version: '2012-10-17'
```

Comment:
Now that I think I wonder how these tests were passing without `Version` before? 
Isn't our actual code generates cfn template that has `Version`? interesting...